### ISSUE TITLE
GraphQL Query date filtering using date not working

### ISSUE THREAD

**Issue by emahuni** [strapi/strapi#3802]
**Describe the bug**
Date filtering through graphql not working as expected. 

**Steps to reproduce the behavior**
1. Query the user model with the below graphql query
2. You should get an empty result

**Expected behavior**
Users that were created after the specified date in the query.

**Code snippets**
For example (from the documentation): 
Return the users which have been created after the March, 19th 2018 4:21 pm.
```graphql
query {
  users(where: {
    created_at_gt: "2018-03-19 16:21:07.161Z"
  }) {
    username
    email
  }
}
```

**Pliz note that I fixed the createdAt_gt field to created_at_gt**, this is not the issue here.

It just returns:
```js
users: []
```

But if I use a unix timestamp eg: `1565474400` multiplied by `1000` it works:

```graphql
query {
  users(where: {
    created_at_gt: 1565474400000
  }) {
    username
    email
  }
}
```

**System**
- Node.js version: 12.6.1
- Strapi version: strapi 3.0.0-beta.15
- Database: SQLite
- Operating system: MacOS Majove

**Comment by derrickmehaffy** [comment]
This has been fixed here: https://github.com/strapi/strapi/pull/3728
And merged into the develop branch, these fixes will be in beta.16 once this PR is merged: https://github.com/strapi/strapi/pull/3664

Marking as closed as the fix has been made but not released yet (other issues are related to this already)
